### PR TITLE
Add Ukrainian Translation

### PR DESCRIPTION
**Pull request by KR1470R**
Add Ukrainian translation.

### CODE REVIEW

File: res/lang/ua.ini [hunk 1]
```diff
@@ -0,0 +1,45 @@
+capslock = capslock
```
Comment (AnErrupTion) [inline res/lang/ua.ini:1]: Please translate this as well.

File: res/lang/ua.ini [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+err_perm_user = не вдалося понизити права доступу користувача
+err_pwnam = не вдалося отримати дані користувача
+err_user_gid = не вдалося змінити GID користувача
+err_user_init = не вдалося ініціалізувати користувача
+err_user_uid = не вдалося змінити UID користувача
+err_xsessions_dir = не вдалося знайти теку сесій
+err_xsessions_open = не вдалося відкрити теку сесій
+f1 = F1 вимкнути
+f2 = F2 перезавантажити
+login = логін:
```
Comment (AnErrupTion) [inline res/lang/ua.ini:39]: Please remove this semicolon.

File: res/lang/ua.ini [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+logout = вийти
+numlock = numlock
+password = пароль:
```
Comment (AnErrupTion) [inline res/lang/ua.ini:42]: Please remove this semicolon.

File: res/lang/ua.ini [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+err_user_gid = не вдалося змінити GID користувача
+err_user_init = не вдалося ініціалізувати користувача
+err_user_uid = не вдалося змінити UID користувача
+err_xsessions_dir = не вдалося знайти теку сесій
+err_xsessions_open = не вдалося відкрити теку сесій
+f1 = F1 вимкнути
+f2 = F2 перезавантажити
+login = логін:
+logout = вийти
+numlock = numlock
```
Comment (AnErrupTion) [inline res/lang/ua.ini:41]: Please translate this as well.

File: res/lang/ua.ini [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+password = пароль:
+shell = shell
```
Comment (AnErrupTion) [inline res/lang/ua.ini:43]: Please translate this as well.

**Review by AnErrupTion — CHANGES_REQUESTED** [review]

File: res/lang/ua.ini (hunk 1 above)
Comment (KR1470R) [reply res/lang/ua.ini:1]: it doesn't translate to ukrainian as well

File: res/lang/ua.ini (hunk 4 above)
Comment (KR1470R) [reply res/lang/ua.ini:41]: it doesn't translate to ukrainian as well

**Comment by KR1470R** [comment]
@AnErrupTion hey! i've pushed fixes, can u merge it pls?

**Comment by AnErrupTion** [comment]
Sorry I took this long! Will merge.

**Pull request merged**